tests/maintel/test_setup_whitelight_flats.py

```diff
@@ -0,0 +1,148 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import os
+import unittest
+
+from lsst.ts import externalscripts, salobj, standardscripts, utils
+from lsst.ts.externalscripts.maintel.setup_whitelight_flats import SetupWhiteFlats
+from lsst.ts.xml.enums import Script
+
+index_gen = utils.index_generator()
+
+
+class TestSetupWhiteFlats(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    def setUp(self):
+        self.log = logging.getLogger(__name__)
+        self.log.propagate = True
+
+    async def basic_make_script(self, index):
+        self.log.debug("Starting basic_make script")
+        self.script = SetupWhiteFlats(index=index)
+
```

Comment:
you should probably call the mock method here to ensure they MTCalsys is mocked every time.